test/plugins/inlineStyles.15.svg

```diff
@@ -0,0 +1,67 @@
+<?xml version="1.0" standalone="no"?>
```

Comment:
FYI, I get some annoying warnings when I run this test. Not sure if it can be avoided.

```
WARN: 'Warning: Syntax error when trying to select 

html /deep/ [hidden]

WARN: 'Warning: Syntax error when trying to select 

html /deep/ [block]

, skipped. Error details: SyntaxError: Unmatched selector: /deep/ [block]'
```

Reply:
@alexjlockwood: The console warnings have been commented out in recent master commits.

Reply:
Yeah, I thought they're more related to dev then user.